src/scripting/floating_image.cpp

```diff
@@ -119,6 +119,6 @@ FloatingImage::fade_out(float time)
   object.fade_out(time);
 }
 
-} // scripting
+} // Scripting.
```

Comment:
This indicates the end of a namespace, should rather be `// namespace scripting`.